index.html

```diff
@@ -0,0 +1,363 @@
+<!doctype html>
+<html class="no-js" lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Morgan Della | Portfolio</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <meta property="og:title" content="">
+  <meta property="og:type" content="">
+  <meta property="og:url" content="">
+  <meta property="og:image" content="img/headshot.jpg">
+
+  <link rel="manifest" href="site.webmanifest">
+  <link rel="apple-touch-icon" href="icon.png">
+  <!-- Place favicon.ico in the root directory -->
+
+  <link rel="stylesheet" href="styles/style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:wght@700&family=Open+Sans:wght@400;700&display=swap" rel="stylesheet">
+
+  <meta name="theme-color" content="#fafafa">
+  <script src="https://kit.fontawesome.com/f38a609730.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <!-- Add your site or application content here -->
+
+  <header>
+    <div class="content-wrapper">
+      <nav>
+        <ul>
+          <li>
+            <a href="#about">About</a>
+          </li>
+          <li>
+            <a href="#skills">Skills</a>
+          </li>
+          <li>
+            <a href="#portfolio">Work</a>
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <div class="header-divider"></div>
+    <section class="greeting">
+      <div class="content-wrapper">
+        <h1><i class="fas fa-sun"></i> Hey y'all!<br>My name is     
+          <span>Morgan Della.</span></h1>
+        <h2>I am a Web Developer with an appreciation for clean, modern web design.</h2>
```

Comment:
In semantic HTML, heading tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)